test/include/util/storage_test_util.h

```diff
@@ -338,6 +353,38 @@ class StorageTestUtil {
     return result;
   }
 
+  static bool SqlTableEqualDeep(const storage::BlockLayout &layout, common::ManagedPointer<storage::SqlTable> table_one,
```

Comment:
Take in two transaction managers and remove the begin transaction hack.

Reply:
Done, good call.